app/renderer/components/navigation/navigationBar.js

```diff
@@ -144,7 +139,8 @@ class NavigationBar extends React.Component {
     props.showHomeButton = !props.titleMode && getSetting(settings.SHOW_HOME_BUTTON)
 
     // used in other functions
-    props.navbar = navbar // TODO(nejc) remove, primitives only
+    props.isFocused = navbar.getIn(['urlbar', 'focused'])
```

Comment:
should this have a default (in case null, etc)? ex:
`props.isFocused = navbar.getIn(['urlbar', 'focused'], false)`